docs/plan/2026-04-25-remote-phase1a-core-contract-implementation.md

```
@@ -0,0 +1,28 @@
+# remote Phase 1A core 契約実装計画
+
+## 目的
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)